resources/docs/livewire/installation.md

```diff
@@ -0,0 +1,176 @@
+
+```ini
+DB_CONNECTION=sqlite
+```
+
+### Installation via Docker
+
+If you do not have PHP installed locally, you may develop your application using [Laravel Sail](https://laravel.com/docs/sail), a light-weight command-line interface for interacting with Laravel's default Docker development environment, which is compatible with all operating systems. Before we get started, make sure to install [Docker](https://docs.docker.com/get-docker/) for your operating system. For alternative installation methods, check out our full [installation guide](https://laravel.com/docs/installation).
+
+The easiest way to install Laravel is using our `laravel.build` service, which will download and create a fresh Laravel application for you. Launch a terminal and run the following command:
+
+```shell
+curl -s "https://laravel.build/chirper" | bash
+```
+
+Sail installation may take several minutes while Sail's application containers are built on your local machine.
+
+By default, the installer will pre-configure Laravel Sail with a number of useful services for your application, including a MySQL database server. You may [customize the Sail services](https://laravel.com/docs/installation#choosing-your-sail-services) if needed.
+
+After the project has been created, you can navigate to the application directory and start Laravel Sail:
+
+```shell
+cd chirper
+
+./vendor/bin/sail up
+```
+
+> **Note**
+> You can [create a shell alias](https://laravel.com/docs/sail#configuring-a-shell-alias) that allows you execute Sail's commands more easily.
+
+ When developing applications using Sail, you may execute Artisan, NPM, and Composer commands via the Sail CLI instead of invoking them directly:
+
+```shell
+./vendor/bin/sail php --version
+./vendor/bin/sail artisan --version
+./vendor/bin/sail composer --version
+./vendor/bin/sail npm --version
+```
+
+Once the application's Docker containers have been started, you can access the application in your web browser at: [http://localhost](http://localhost).
+
+<img src="/img/screenshots/fresh.png" alt="A fresh Laravel installation" class="dark:hidden rounded-lg border shadow-lg" />
+<img src="/img/screenshots/fresh-dark.png" alt="A fresh Laravel installation" class="hidden dark:block rounded-lg border-gray-700 shadow-lg" />
+
+## Installing Livewire
+
+Now that we have a fresh Laravel application set up, we can install both Livewire and Livewire Volt. The latter is a simple Livewire extension that allows a Livewire component's PHP logic and Blade templates to coexist in the same file, similar to modern JavaScript frameworks like Vue and React.
+
+Open a new terminal in your `chirper` project directory and install Livewire with the given command:
+
+```shell
+composer require livewire/livewire livewire/volt
+
+php artisan volt:install
+```
+
+## Installing Laravel Breeze
```

Comment:
It doesn't make a huge difference, but I think Breeze should be installed first (i.e. Before Livewire and Volt) as a best practice.